static/scss/answers/theme.scss

```diff
@@ -67,6 +67,9 @@
         padding-left: var(--yxt-base-spacing);
```

Comment:
How is the DOM in the SDK component structured? Will this style impact all Nav tabs (not just those in the drop-down)?

Reply:
Yes, this styling would affect all Nav tabs. Updated to only change the drop-down items!